### ISSUE TITLE
BTdevice discovery lists duplicate entries

### ISSUE THREAD

**Issue by Norgus** [TheLastProject/ShareToInputStick#41]
The list of discovered devices seems to just keep growing instead of deduplicating already discovered devices.

**Comment by TheLastProject** [comment]
Yeah I've definitely seen this issue before. Thanks for reporting. I'm honestly quite busy with other projects with more users so I don't know when I'll get to it, but it's good to have it tracked in the issue tracker :) It's a bit annoying, but doesn't really seem to harm the app usage too much luckily.
